src/app/models/vehicle-tech-record.model.ts

```diff
@@ -1,11 +1,138 @@
 export interface VehicleTechRecordModel {
-  vrms: VrmModel[];
+  vrms: vrms[];
```

Comment:
nitpick: the array brackets denote plurality, so the 's' can be dropped from the model
```suggestion
  vrms: Vrm[];
```

Reply:
Looks like all of the additional interface declarations are plural, and I'd suggest they should be singular; consider:

`StatusCode.ARCHIVED`
vs
`StatusCodes.ARCHIVED`